admin-ui/app/locales/en/translation.json

```
@@ -347,7 +353,9 @@
     "log_layout": "Log layout",
     "login_uris": "Login URIs",
     "logo_uri": "Logo URI",
+    "logout_uris": "Logout URIs",
     "logout_redirect_uris": "Logout Redirect URIs",
+    "logout_status_jwt_signed_response_alg": "Logout Status JWT Signed Response Alg",
```

Biome (2.1.2)

[error] The key logout_status_jwt_signed_response_alg was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)